WordPress/Sniffs/Theme/SearchFormSniff.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check that no include calls to searchform.php are found,
+ * recommend using get_search_form() instead.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/theme-check-plugin/#admin-menu
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_SearchFormSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * Returns an array of tokens this test wants to listen for.
+	 *
+	 * @return array
+	 */
+	public function register() {
+		return array(
+			T_STRING,
+			T_CONSTANT_ENCAPSED_STRING,
+		);
+	}
+
+	/**
+	 * Processes this test, when one of its tokens is encountered.
+	 *
+	 * @param PHP_CodeSniffer_File $phpcsFile The file being scanned.
+	 * @param int                  $stackPtr  The position of the current token
+	 *                                        in the stack passed in $tokens.
+	 *
+	 * @return void
+	 */
+	public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
+
+		$tokens = $phpcsFile->getTokens();
+		$token  = $tokens[ $stackPtr ];
+
+		if ( false !== strpos( trim( $token['content'] . '\"\'' ), 'searchform.php' ) ) {
```

Comment:
Three things here:
1. `trim( $token['content'] . '\"\'' )` => `.` (dot = concatenation) should be a `,` (comma = next parameter indicator).
2. Similar to the include/require sniff, it might be better to create a (array) class property to hold the `searchform.php` to allow for more files to be added in the future. As a value for the array I'd suggest using the alternative `get_search_form()`.
3. In that case, this line will have to be revisited again depending on how the property is set up.